client/app/scripts/liveblog-edit/directives.js

```diff
@@ -29,8 +29,11 @@ define([
                     angular.extend(vm, {
                         isLoading: true,
                         blogId: $scope.lbPostsBlogId,
-                        allowUnpublish: $scope.lbPostsAllowUnpublish,
-                        allowReordering: $scope.lbPostsAllowReordering,
+                        allowUnpublishing: $scope.lbPostsAllowUnpublishing || false,
```

Comment:
it sound reasonable if you would check against boolean type, if not they don't have any meaning and it just makes the code harder to read.